README.md

```diff
@@ -11,29 +11,84 @@ For general project information, visit the [Harmony wiki](https://wiki.earthdata
     1. [Earthdata Login Application Requirement](#Earthdata-Login-Application-Requirement)
     2. [Software Requirements](#Software-Requirements)
-2. [Running Harmony](#Running-Harmony)
+4. [Running Harmony](#Running-Harmony)
     1. [Set Up Environment Variables](#Set-Up-Environment-Variables)
     2. [Run Tests](#Run-Tests)
     3. [Set Up A Database](#Set-Up-A-Database)
     4. [Set Up and Run Argo, Localstack](#Set-Up-and-Run-Argo,-Localstack)
     5. [Add A Service Backend](#Add-A-Service-Backend)
     6. [Run Harmony](#Run-Harmony)
     7. [Connect A Client](#Connect-A-Client)
-3. [Local Development Of Workflows Using Visual Studio Code](#Local-Development-Of-Workflows-Using-Visual-Studio-Code)
-4. [Running in AWS](#Running-in-AWS)
-5. [Contributing to Harmony](#Contributing-to-Harmony)
-6. [Additional Resources](#Additional-Resources)
+5. [Local Development Of Workflows Using Visual Studio Code](#Local-Development-Of-Workflows-Using-Visual-Studio-Code)
+6. [Running in AWS](#Running-in-AWS)
+7. [Contributing to Harmony](#Contributing-to-Harmony)
+8. [Additional Resources](#Additional-Resources)
+
+## Minimum System Requirements
+
+* A running [Docker Desktop](https://www.docker.com/products/developer-tools) or daemon instance - Used to invoke docker-based services
+* A running [Kubernetes](https://kubernetes.io/) cluster with the [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) command. [Docker Desktop](https://www.docker.com/products/docker-desktop) for Mac and Windows comes with a
+built-in Kubernetes cluster (including `kubectl`) which can be enabled in preferences.
+
+## Quick Start
+(Mac OS X / Linux)
+
+If you are interested in using a local Harmony instance to develop services, but not interested in 
+developing the Harmony code itself, the following steps are enough to start a locally running Harmony instance.
+
+1. Follow the directions for creating an Earth Data Login application and credentials in the [Earthdata Login Application Requirement](#Earthdata-Login-Application-Requirement) section below.
+
+2. Download this repository (or download the zip file from GitHub)
+```bash
+git clone https://github.com/nasa/harmony.git
+```
+3. Run the bootstrap script and answer the prompts with your EDL application credentials
+```bash
+./bin/bootstrap-harmony
+```
+
+Harmony should now be running in your Kubernetes cluster as the `harmony` service in the `argo` namespace. If you installed
+the example harmony service you can test it with the following (requires a [.netrc](https://www.gnu.org/software/inetutils/manual/html_node/The-_002enetrc-file.html) file):
+
+```bash
+curl -Ln -bj "http://localhost:3000/C1233800302-EEDTEST/ogc-api-coverages/1.0.0/collections/all/coverage/rangeset?granuleId=G1233800343-EEDTEST" -o file.tif
+```
+You can view the workflow running in [Argo](https://argoproj.github.io/projects/argo) by opening the Argo UI at [http://localhost:2746](http://localhost:2746).
```

Comment:
May end by recommending use of harmony-py and its notebook?

Reply:
Done.